components/admin_console/system_user_detail/team_list/team_list_dropdown.tsx

```diff
@@ -10,23 +9,27 @@ import EllipsisHorizontalIcon from 'components/widgets/icons/ellipsis_h_icon';
 import Menu from 'components/widgets/menu/menu';
 import MenuWrapper from 'components/widgets/menu/menu_wrapper';
 
-export default class TeamListDropdown extends React.Component {
-    static propTypes = {
-        team: PropTypes.object.isRequired,
-        doRemoveUserFromTeam: PropTypes.func.isRequired,
-        doMakeUserTeamAdmin: PropTypes.func.isRequired,
-        doMakeUserTeamMember: PropTypes.func.isRequired,
-    }
+type Props = {
+    team: {[x: string]: string};
+    doRemoveUserFromTeam: (teamId: string) => Promise<void>;
```

Comment:
Non-blocking, but since this component doesn't use the return value of these functions, we could just define their return type as `void`, since there's no `Promise` requirement here.

Reply:
Just did this.